scripts/create-a-m-o/index.js

```diff
@@ -0,0 +1,87 @@
+const outdent = require('outdent');
+const chalk = require('chalk');
+const readline = require('readline');
+
+// const {
+//   getAMOType,
+// } = require('./functions.js');
```

Comment:
Please remove this if not needed.

Reply:
As it says in the PR description, this is WIP. At the end i wont have commented code